README.md

```diff
@@ -1,75 +1,62 @@
+* We encourage direct access of react-router APIs. Need server-side rendering, or something else advanced? Just read react-router's docs.
+* We only store current URL and state, whereas redux-router stores the entire location object from react-router.
 
-[redux-router](https://github.com/rackt/redux-router) is another
-project which solves the same problem. However, it's far more complex.
-Just look at this code: the whole thing is only 68 lines of JS.
-redux-router is much bigger and more complex.
+This library only depends on the `history.listen` function from react-router and the `store.getState` and `store.subscribe` functions from redux. It should be very future-proof with any versions of either libraries.
 
-That said, redux-router is a fine project and has features this
-doesn't provide. Use it if you like it better.
+## <a id="docs" href="#how-to-use">Documentation</a>
 
-The differences between this and redux-router:
+0. [Usage](#how-to-use)
+ - [Installation](#installation)
+0. [Tutorial](#tutorial)
+0. [Examples](#examples)
+0. [API](#api)
+ - [`syncReduxAndRouter(history, store, selectRouterState?)`](#syncreduxandrouter)
+ - [`routeReducer`](#routereducer)
+ - [`UPDATE_PATH`](#update-path)
+ - [`pushPath(path, state, { avoidRouterUpdate = false } = {})`](#pushpath)
+ - [`replacePath(path, state, { avoidRouterUpdate = false } = {})`](#replacepath)
+0. [See also](#see-also)
 
-* Much smaller and simpler. You don't need to learn another library on
-  top of everything else.
-* We encourage direct access of react-router APIs. Need server-side
-  rendering, or something else advanced? Just read react-router's
-  docs.
-* The piece of state is just a URL string, whereas redux-router stores
-  the full location object from react-router.
+### <a id="how-to-use" href="#how-to-use">Usage</a>
 
-This only depends on the `history.listen` function from react-router
-and the `store.getState` and `store.subscribe` functions from redux.
-It should be very future-proof with any versions of either libraries.
+The idea of this library is to use react-router's functionality exactly like its documentation tells you to. You can access all of its APIs in routing components. Additionally, you can use redux like you normally would, with a single app state and "connected" components. It's even possible for a single component to be both if needed.
 
-## Examples
+[redux](https://github.com/rackt/redux) (`store.routing`) &nbsp;&harr;&nbsp; [**redux-simple-router**](https://github.com/jlongster/redux-simple-router) &nbsp;&harr;&nbsp; [history](https://github.com/rackt/history) (`history.location`) &nbsp;&harr;&nbsp; [react-router](https://github.com/rackt/react-router)
 
-There are examples in the `examples` directory:
+This library provides a single point of synchronization: the `routing.path` piece of state which is the current URL. You can read it, and also change it with an action.
```

Comment:
"This library provides a single point of synchronization" is no longer true. There's also `routing.state` now.